doc/POST_INSTALL.md

```diff
@@ -1,7 +1,7 @@
 ## Configuration
```

Comment:
All this file should be reworked, 
- the LDAP thing is supposed to go in the manifest (though I guess we can keep a note in ADMIN.md maybe), 
- architectures go in the manifest too, 
- `How to configure this app: From an admin panel, a plain file with SSH.` << I have no idea what this means x_x
- `How to change from edition version to another (Team<->Enterprise)` : this should probably go in ADMIN.md
- `YunoHost specific features` : probably to move to the DESCRIPTION.md